concent_api/core/utils.py

```diff
@@ -124,22 +124,28 @@ def extract_name_from_scene_file_path(absoulte_scene_file_path_in_docker: str) -
     return relative_scene_file_path_in_archive
 
 
+def is_protocol_verison_compatible_with_verison_in_concent(protocol_version: str) -> bool:
+    """
+    Versions are considered compatible if they share the minor and major version number.
+    E.g 2.18.5 is compatible with 2.18.1 but not with 2.17.5 or 3.0.0.
+    """
+    return protocol_version.split('.')[0] == settings.GOLEM_MESSAGES_VERSION.split('.')[0] and \
+        protocol_version.split('.')[1] == settings.GOLEM_MESSAGES_VERSION.split('.')[1]
+
+
 def if_given_version_of_golem_messages_is_compatible_with_version_in_concent(
     request: HttpRequest,
     client_public_key: bytes,
 ) -> bool:
     """
-    Versions are considered compatible if they share the minor and major version number.
-    E.g 2.18.5 is compatible with 2.18.1 but not with 2.17.5 or 3.0.0. If header is missing
-    version is not checked and Concent assumes that client uses compatible version.
```

Comment:
I see that this part of code is changed in this PR 3rd time. It wouldn't be better if it be done in one commit?

Reply:
It would be, but as I said: in one commit I created function to solve one problem. In other commit I solved similar problem and had to change this function to reuse it and avoid code duplication, etc. In first commit I haven't know final version of this function. BTW. even if I would have it would be wrong because of overkill. In PR there is logic way to solve the problem, if some refactor was required- it was